auto_nag/scripts/topcrash_bad_severity.py

```diff
@@ -15,7 +15,16 @@ def ignore_date(self):
     def get_bz_params(self, date):
         return {
             "resolution": ["---"],
-            "bug_severity": ["major", "normal", "minor", "trivial", "enhancement"],
+            "bug_severity": [
+                "S2",
```

Comment:
Maybe topcrash could be S2 and not S1 in some cases (given the definition in https://firefox-source-docs.mozilla.org/bug-mgmt/guides/severity.html). Could you run this script in dry mode and post the result here?

Reply:
Here is  the dry run query: https://mzl.la/3KRVNsT

Reply:
Can a bug be `topcrach` without the possibility of affecting 25% of the users?

Reply:
Yes, I think most top crashes are only affecting a small subset of the population, even though they are "top".
Given the list in https://mzl.la/3KRVNsT, we should remove "S2" from the list and allow top crashes to be S2.

Reply:
For consistency, I think we should remove `major` as well, which translates to `S2`. Wdyt?

Reply:
The old severities were a bit of a mess, sometimes "critical" corresponded to "S1", sometimes it corresponded to "S2", sometimes it was not even "S1" or "S2" because it was set by some random user. The situation is not perfect now, but at least a little better.
It doesn't matter much in the end anyway because these old values are no longer used, so feel free to do as you prefer!